der/src/asn1/integer/int.rs

```diff
@@ -34,14 +46,14 @@ where
     writer.write(strip_leading_ones(bytes))
 }
 
-/// Get the encoded length for the given unsigned integer serialized as bytes.
+/// Get the encoded length for the given signed integer serialized as bytes.
 #[inline]
 pub(super) fn encoded_len(bytes: &[u8]) -> Result<Length> {
     Length::try_from(strip_leading_ones(bytes).len())
 }
 
 /// Strip the leading all-ones bytes from the given byte slice.
-fn strip_leading_ones(mut bytes: &[u8]) -> &[u8] {
+pub(crate) fn strip_leading_ones(mut bytes: &[u8]) -> &[u8] {
```

Comment:
N.B.: I updated these comments; based on the context and behavior it looks like they were incorrectly copy-pasted before.